Update api-status-codes.md (#1366)
bigcommerce/dev-docs: docs/api-docs/getting-started/api-status-codes.md

# API Status Codes

> The BigCommerce API responds to a request with different HTTP status codes depending on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.

Reviewer: ```suggestion
The BigCommerce API responds to each request with an HTTP status code that depends on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.

```

Reviewer: @bc-tgomez -- I like the shift to singular request.  How do you feel about this?

| Code | Text | Purpose |
|-|-|-|
| **400** | **Bad Request** | Issued when a malformed request was sent. |

> ||| The request can not be completed due to a URL restriction. |

Reviewer 2: ```suggestion
||| The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |

```

| Code | Text | Purpose |
|-|-|-|
| **400** | **Bad Request** | Issued when a malformed request was sent. |

> ||| The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |

Reviewer 3: When I preview this doc, there's something off about the table formatting. Can you take a look again?

Author: Definitely, I'll take a look at this.

Author: Fixed? Maybe? Preview looks better.
